storey/writers.py

```diff
@@ -554,12 +554,21 @@ def __init__(self, table: Union[Table, str], columns: Optional[List[str]] = None
         self._closeables = [self._table]
 
     def _init(self):
-        _ConcurrentByKeyJobExecution._init(self)
         _Writer._init(self)
 
-    async def _process_events(self, events):
-        data_to_persist = self._event_to_writer_entry(events[-1])
-        return await self._table._persist_key(events[0].key, data_to_persist)
-
     async def _handle_completed(self, event, response):
         await self._do_downstream(event)
+
+    async def _do(self, event):
+        if event is _termination_obj:
+            await self._table._terminate()
+            return await self._do_downstream(_termination_obj)
+        elif self._table._flush_interval_milli == 0:
```

Comment:
Why is flush interval also used by WriteToTable? I thought it's for the table to persist by itself, not a parameter to WriteToTable.

Reply:
It seems to me that WriteToTable should persist all events it gets, which is in turn controlled by the emit policy (so whether it will persist for each event or not is controlled by that already).